### PR TITLE
Add logos feature updated for declarative plotting changes

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
Addresses issue #46. 

Adds option to plot either just NOAA logo, just NWS logo, or both. Plots logo on all axes.

image

### CODE REVIEW

File: src/emcpy/plots/create_plots.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
 # This work developed by NOAA/NWS/EMC under the Apache 2.0 license.
 import os
+import emcpy
```
Comment (CoryMartin-NOAA) [inline src/emcpy/plots/create_plots.py:3]: why are we importing emcpy wholesale here? Should do like what is done at the end of the `import` list

Comment (kevindougherty-noaa) [reply src/emcpy/plots/create_plots.py:3]: Good question. I was taking a look at how the old version got the path needed to get to the logo `.png` files, and it had `emcpy.emcpy_directory` (line 333), so used that. Is there a better approach to point to the directory with the logo figures?

Comment (CoryMartin-NOAA) [reply src/emcpy/plots/create_plots.py:3]: then maybe just do something like `import emcpy.emcpy_directory as emcpydir` or something like that?

File: src/emcpy/plots/create_plots.py [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 import numpy as np
 import matplotlib
 import matplotlib.pyplot as plt
 import matplotlib.gridspec as gridspec
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
+from PIL import Image
```
Comment (danholdaway) [inline src/emcpy/plots/create_plots.py:10]: Is this a new dependency or something an existing dependency already depends on?

**Review by danholdaway — APPROVED** [review]
Cool change!

File: src/emcpy/plots/create_plots.py (hunk 2 above)
Comment (CoryMartin-NOAA) [reply src/emcpy/plots/create_plots.py:10]: good question, I know it is 'wcoss approved' but not sure if we already have it in our env or not... (seems we do if the tests pass)

**Comment by danholdaway** [comment]
It seems we're in a space now where you're adding things to EMCpy not in Eva and I'm making changes to the EMCpy part of Eva that aren't in EMCpy. Should we put our heads together next week to converge?

File: src/emcpy/plots/create_plots.py (hunk 2 above)
Comment (danholdaway) [reply src/emcpy/plots/create_plots.py:10]: Might possibly need to be added to the setup.py of eva. Maybe PIL is synonymous with pillow? Pillow is already satisfied by other packages.

**Comment by CoryMartin-NOAA** [comment]
@danholdaway sorry yes, we need to yank the plotting pieces (largely) out of eva soon/now. This was poor coordination on my/our (EMC's) fault.

File: src/emcpy/plots/create_plots.py (hunk 2 above)
Comment (CoryMartin-NOAA) [reply src/emcpy/plots/create_plots.py:10]: yeah it's pillow

**Pull request merged**